<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color :'white'</string>
   </property>
   <widget class="QLabel" name="AppName">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color : rgb(118, 173, 232);
border-radius :50px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Auto Whatsapp&lt;/p&gt;&lt;p align=&quot;center&quot;&gt; Broadcast&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PetunjukPenggunaan">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color : rgb(177, 220, 242);
border-radius : 25px;
}</string>
    </property>
    <property name="text">
     <string>Usage Guidance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BroadcastTanpaDokumen">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color : rgb(177, 220, 242);
border-radius : 25px;
}</string>
    </property>
    <property name="text">
     <string>Broadcast Without Attachment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BroadcastDenganDokumen">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>550</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color : rgb(177, 220, 242);
border-radius : 25px;
}</string>
    </property>
    <property name="text">
     <string>Broadcast With Attachment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FullProjectLink">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>720</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color : rgb(177, 220, 242);
border-radius : 25px;
}</string>
    </property>
    <property name="text">
     <string>Credit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
